modules/LBSTransientSolver/IterativeOperations/lbts_settransientsource.cc

```diff
@@ -70,157 +70,125 @@ void lbs::TransientSolver::
     if (P0_src and apply_fixed_src)
       src = P0_src->source_value_g.data();
 
-    //=========================================== Loop over nodes
+    //======================================== Loop over nodes
     const int num_nodes = transport_view.NumNodes();
     for (int i = 0; i < num_nodes; ++i)
     {
-      //==================================== Loop over moments
+      //======================================== Loop over moments
       for (int m = 0; m < static_cast<int>(num_moments); ++m)
       {
         unsigned int ell = m_to_ell_em_map[m].ell;
 
         size_t uk_map = transport_view.MapDOF(i, m, 0); //unknown map
 
-        //============================= Loop over groupset groups
+        //=================================== Loop over groupset groups
         for (size_t g = gs_i; g <= gs_f; ++g)
         {
-          if (not options.use_src_moments) //using regular material src
-          {
-            if (apply_fixed_src and ell == 0)
-              destination_q[uk_map + g] += src[g];
-          }
+          double rhs = 0.0;
+
+          //============================== Apply fixed sources
+          if (!options.use_src_moments) //using regular material src
+            rhs += apply_fixed_src && ell == 0? src[g] : 0.0;
```

Comment:
I would add a space before the ? for readability

Reply:
I've always used `cond? true : false` and sometimes `(cond)? true : false`. We should see how these are written in the rest of the code.